playground/yarp/Yarp.AppHost/Program.cs

```diff
@@ -10,6 +10,7 @@
 var frontendService = builder.AddProject<Projects.Yarp_Frontend>("frontend");
 
 var gateway = builder.AddYarp("gateway")
+                     .WithStaticFiles() // Enable static file serving
```

Comment:
Add another yarp instance to the example, add static files and bindmount an index.html file into it /wwwroot of the container.